File: src/classes/api/req_res/upload-files-response.ts
// ˅
'use strict';

import { Kyou } from '@/classes/datas/kyou';
import { GkillAPIResponse } from '../gkill-api-response';

// ˄

export class UploadFilesResponse extends GkillAPIResponse {
    // ˅
    
    // ˄

    uploaded_kyous: Array<Kyou>;

    constructor() {
        // ˅
        super()
        this.uploaded_kyous = new Array<Kyou>()
        // ˄
    }

    // ˅
    
    // ˄
}

// ˅

// ˄

File: src/app/gkill/api/req_res/upload_files_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/app/gkill/api/message"
	"github.com/mt3hr/gkill/src/app/gkill/dao/reps"
)

type UploadFilesResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	UploadedKyous []reps.Kyou `json:"uploaded_kyous"`
}
